libgambatte/libretro/libretro_core_options.h

```diff
@@ -624,9 +624,9 @@ struct retro_core_option_v2_definition option_defs_us[] = {
    },
    {
       "gambatte_rumble_level",
-      "Gamepad Rumble Strength",
+      "Controller Rumble Strength",
       NULL,
-      "Enables haptic feedback effects for supported games (Pokemon Pinball, Perfect Dark...).",
+      "Enables haptic feedback effects for supported games (Pokémon Pinball, Perfect Dark...).",
```

Comment:
We prefer to avoid using extended ASCII in the US English core options text, since by default this won't display correctly when using RGUI

Reply:
Pure ASCII for the English texts - got it!